pkg/audit/verifier.go

```diff
@@ -170,6 +170,9 @@ func (d *defaultDownloader) getShare(ctx context.Context, limit *pb.AddressedOrd
 	timedCtx := ctx
 	if d.minBytesPerSecond > 0 {
 		maxTransferTime := time.Duration(int64(time.Second) * int64(bandwidthMsgSize) / d.minBytesPerSecond.Int64())
```

Comment:
In release-alpha7 we fixed the issue with https://github.com/storj/storj/blob/0bacce74d3e1f7c6016503595acbd4704c712720/pkg/audit/verifier.go#L172

As far as I understand the important part is time.Seconds * time.Duration. On master we now try the old code that was producing wrong values? Please don't do that.

Reply:
@littleskunk Did you tests / resolve this difference?

Reply:
Yes I have tested it. The result looks good.